### ISSUE TITLE
Not able to customize machina template and classes

### ISSUE THREAD

**Issue by snehlata08** [ellmetha/django-machina#244]
I am following  [this tutorial](https://django-machina.readthedocs.io/en/stable/customization/index.html) to customise machina. 

`
import os
TEMPLATES = [
    {
        'BACKEND': 'django.template.backends.django.DjangoTemplates',
        'DIRS': [
          os.path.join(PROJECT_PATH, 'myproject/templates'),
              MACHINA_MAIN_TEMPLATE_DIR,
        ],
        'OPTIONS': {
          'context_processors': [
             # [...]
          ],
        },
      },
    ]
`

I have changed the path to my machina project but still the template changes are not reflecting. 
As well as I did the following to customise the machina code 
`$ mkdir -p apps/forum_conversation
$ touch apps/__init__.py
$ touch apps/forum_conversation/__init__.py`

But my class changes were not visible. So can anyone tell me what am i doing wrong over here.

**Comment by AliIslamov** [comment]
I also can't understand this problem. Very poor documentation 😒

**Comment by ellmetha** [comment]
> I have changed the path to my machina project but still the template changes are not reflecting.

Regarding customizing templates, if you followed the [dedicated guidelines](https://django-machina.readthedocs.io/en/stable/customization/index.html#templates-and-static-files), you should be able to define overrides of any django-machina template. For example, I just tested this with the [example project](https://github.com/ellmetha/django-machina/tree/main/example_project), which defines a [templates configuration similar to yours](https://github.com/ellmetha/django-machina/blob/main/example_project/project/settings/base.py#L139-L145), and I was able to define a `forum_conversation/topic_create.html` file that was overriding the built-in template as expected. The path to the overridden templates (in your templates folder) must not include the `machina` folder for this to work properly.

> As well as I did the following to customise the machina code

Have you followed the [Overriding an application](https://django-machina.readthedocs.io/en/stable/customization/overriding_applications.html) tutorial up to the end? I am asking because the commands you are mentioning refer to the first step of this tutorial only.

> Very poor documentation 😒

@AliIslamov I welcome any feedback on this project, but I kindly request that it be provided in a respectful and positive manner. A significant amount of effort went into creating the documentation you've referred to as 'poor,' and while constructive criticism is valuable, let's maintain a collaborative and positive atmosphere here.

**Comment by AliIslamov** [comment]
> @AliIslamov I welcome any feedback on this project, but I kindly request that it be provided in a respectful and positive manner. A significant amount of effort went into creating the documentation you've referred to as 'poor,' and while constructive criticism is valuable, let's maintain a collaborative and positive atmosphere here.

Yes, your are absolutely right. I'm sorry for my unconstructive overly emotional statement.

You already have done great app and I purchased your for it.

I just want to say, that it will be very helpful if you do more examples in docs.

Beginners like me really need this.

Best regards.
